stopwatch-NGVM/prompts.md

```
@@ -0,0 +1,14 @@
+https://chatgpt.com/share/68e05bb4-2064-800e-bcf5-8071fa7765ad
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)